sdk/appconfiguration/test-resources.json

```diff
@@ -0,0 +1,39 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "baseName": {
+            "type": "string",
+            "defaultValue": "[resourceGroup().name]",
```

Comment:
Did we not want to force baseName to get value from outside as random value ? The default value means if we do not get random value, it will create resource.

Reply:
I can change it!

But we should probably leave that feedback in Heath's PR: https://github.com/Azure/azure-sdk-for-net/pull/9029/files#diff-6811108f0ad7825011166ac5bd031bc3R5-R11